account_financial_risk/views/portal_templates.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Tecnativa - Pilar Varga
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="portal_layout" inherit_id="portal.portal_layout">
+        <xpath expr="//div[hasclass('o_portal_my_security')]" position="after">
+            <div
+                t-if="user_id.partner_id.portal_show_financial_risk_visible and user_id.partner_id.show_financial_risk_in_portal and user_id.partner_id.sudo().credit_limit != 0"
```

Comment:
```suggestion
                t-if="user_id.partner_id.portal_show_financial_risk_visible and user_id.partner_id.show_financial_risk_in_portal and user_id.partner_id.sudo().commercial_partner_id.credit_limit != 0"
```
It might be useful to define a variable (`partner_financial_risk = user_id.partner_id.sudo().commercial_partner_id`) and use it everywhere.

Reply:
Good tip! thanks!